bundles/framework/layerlist/Tile.js

```diff
@@ -0,0 +1,110 @@
+        /**
+         * @method getName
+         * @return {String} the name for the component
+         */
+        getName: function () {
+            return 'Oskari.mapframework.bundle.layerlist.Tile';
+        },
+        /**
+         * @method setEl
+         * @param {Object} el
+         *     reference to the container in browser
+         * @param {Number} width
+         *     container size(?) - not used
+         * @param {Number} height
+         *     container size(?) - not used
+         *
+         * Interface method implementation
+         */
+        setEl: function (el, width, height) {
+            this.container = jQuery(el);
+        },
+        /**
+         * @method startPlugin
+         * Interface method implementation, calls #refresh()
+         */
+        startPlugin: function () {
+            this._addTileStyleClasses();
+            this.refresh();
+        },
+
+        _addTileStyleClasses: function () {
+            var isContainer = !!((this.container && this.instance.mediator));
+            var isBundleId = !!((isContainer && this.instance.mediator.bundleId));
+            var isInstanceId = !!((isContainer && this.instance.mediator.instanceId));
+
+            if (isInstanceId && !this.container.hasClass(this.instance.mediator.instanceId)) {
+                this.container.addClass(this.instance.mediator.instanceId);
+            }
+            if (isBundleId && !this.container.hasClass(this.instance.mediator.bundleId)) {
+                this.container.addClass(this.instance.mediator.bundleId);
+            }
+        },
+        /**
+         * @method stopPlugin
+         * Interface method implementation, clears the container
+         */
+        stopPlugin: function () {
+            this.container.empty();
+        },
+        /**
+         * @method getTitle
+         * @return {String} localized text for the title of the tile
+         */
+        getTitle: function () {
+            return this.instance.getLocalization('title');
+        },
+        /**
+         * @method getDescription
+         * @return {String} localized text for the description of the tile
+         */
+        getDescription: function () {
+            return this.instance.getLocalization('desc');
+        },
+        /**
+         * @method getOptions
+         * Interface method implementation, does nothing atm
+         */
+        getOptions: function () {
+
+        },
+        /**
+         * @method setState
+         * @param {Object} state
+         *     state that this component should use
+         * Interface method implementation, does nothing atm
+         */
+        setState: function (state) {
+
+        },
+        /**
+         * @method refresh
+         * Creates the UI for a fresh start
+         */
+        refresh: function () {
+        }
+    }, {
+        /**
+         * @property {String[]} protocol
+         * @static
+         */
+        'protocol': ['Oskari.userinterface.Tile']
```

Comment:
If you extend 'Oskari.userinterface.extension.DefaultTile' this would be far fever lines. For now the custom tile isn't needed but we need a custom implementation for the "selected layers count" label that we will add in a future update.

Reply:
Using the DefaultTile directly for now